comp/core/workloadmeta/impl/workloadmeta_mock.go

```diff
@@ -3,8 +3,6 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build test
```

Comment:
Why are you removing this ? It results in the mock ending up in the final binaries

Reply:
ah sorry I just read the response in the other thread, do you know why it ends up in all the binaries and not just core + dogstatsd on the other hand ?

Reply:
No I don't know, AFAIK we don't have any build tags for this?